src/soot/jimple/spark/solver/PropWorklist.java

```diff
@@ -69,6 +69,9 @@ public final void propagate() {
 						public final void visit(Node n) {
 							AllocDotField nDotF = pag.makeAllocDotField(
 									(AllocNode) n, target.getField());
+                            if(ofcg != null) {
```

Comment:
Let's not discuss the coding style in the older parts of Soot, because this won't lead us anywhere. Just stick to what, e.g., the Eclipse code formatter produces or what yu see in newer parts of the code, ad you're fine.